resources/developer-mode.mjs

```diff
@@ -168,7 +193,24 @@ function updateURL() {
     } else {
         url.searchParams.delete("tags");
         url.searchParams.delete("suite");
-        url.searchParams.set("suites", selectedSuites.join(","));
+        // Try finding common tags that would result in the current suite selection.
+        let commonTags = new Set(selectedSuites[0].tags);
+        for (const suite of Suites) {
+            if (suite.disabled) {
+                // commonTags = commonTags.difference(suite.tags);
+                suite.tags.forEach((tag) => commonTags.delete(tag));
+            } else {
+                // commonTags = commonTags.intersection(suite.tags);
```

Comment:
Nit: commented line can be removed

Reply:
I mostly added them as comments since the code is maybe not fully clear (but happy to remove as well).